packages/angular_devkit/build_angular/src/differential-loading/create-webpack-multi-config.ts

```diff
@@ -0,0 +1,185 @@
+/**
```

Comment:
inline comments near the implementations would definitely help the reviewers and future engineers that go through the code, without the need of going back and forth to the design doc.